src/generator/synth.ts

```diff
@@ -132,6 +154,17 @@ export function createChangelog(changeSets: ChangeSet[]) {
   };
 }
 
+export function getPrefix(semverity: Semverity) {
+  switch (semverity) {
+    case Semverity.PATCH:
+      return 'fix';
+    case Semverity.MINOR:
+      return 'feat';
+    case Semverity.MAJOR:
+      return 'feat!';
```

Comment:
the spec dictates that `feat(foo)!` would be how you indicate a breaking change, when both types and feature are used.

Why don't you make this a regex rather than a string comparison, then do something like `/feat(\(.*\))?!/`

Reply:
Great catch.